transport/lwm2m/src/main/resources/tb-lwm2m-transport.yml

```diff
@@ -103,6 +103,10 @@ transport:
     enabled: "${TB_TRANSPORT_LOG_ENABLED:true}"
     max_length: "${TB_TRANSPORT_LOG_MAX_LENGTH:1024}"
   # Local LwM2M transport parameters
+  coap:
```

Comment:
make the same changes as thingsboard.yml

Reply:
Ok